lib/team_api/api_impl.rb

```diff
@@ -32,6 +33,15 @@ def generate_index_endpoint(endpoint, title, description, items)
       }
     end
 
+    def generate_schema_endpoint(schema_file_location)
+      return if schema_file_location.nil? || schema_file_location.empty?
```

Comment:
If we trigger this return statement, what will happen to the URL?  Is it just not generated at all and therefore not available as an endpoint (in other words, would it even be available)?  If not, should we hide the endpoint in that situation so folks can't get to a broken URL?